.goreleaser.yml

```diff
@@ -12,6 +12,17 @@ builds:
       - linux
       - windows
       - darwin
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - 7
+    ignore:
+      - goos: windows
+        goarch: arm
+        goarm: 7
```

Comment:
Why do we ignore this? Could you add a comment maybe to the yaml file?

Reply:
I didn't think there were any current 32bit ARM Windows versions left but apparently [Embedded Compact and IoT Core](https://en.wikipedia.org/wiki/ARM_architecture_family#32-bit_operating_systems) are still supported.

I'll take this out.